Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/__init__.py

```diff
@@ -645,66 +649,94 @@ def add_site_dir(site_dir):
             _LOGGER.exception(f'{e} , traceback =', exc_info=True)
             raise e
 
+_LOGGER.debug(f'Final {ENVAR_PATH_O3DE_3RDPARTY} is: {str(PATH_O3DE_3RDPARTY)}')
+# -------------------------------------------------------------------------
 
-_LOGGER.debug(f'Final {ENVAR_PATH_O3DE_3RDPARTY} is: {PATH_O3DE_3RDPARTY.as_posix()}')
 
+# -------------------------------------------------------------------------
 # first check if we can get from o3de
 ENVAR_QT_PLUGIN_PATH = 'QT_PLUGIN_PATH'
-try:
-    import azlmbr
-    import azlmbr.bus
-    params = azlmbr.qt.QtForPythonRequestBus(azlmbr.bus.Broadcast, 'GetQtBootstrapParameters')
-    QT_PLUGIN_PATH = Path(params.qtPluginsFolder)
-except:
-    QT_PLUGIN_PATH = None
-
-if not QT_PLUGIN_PATH:
-    # fallback, not future proof without editing file
-    # modify to be a grep?
-    # path constructor
-    QT_PLUGIN_PATH = Path(PATH_O3DE_3RDPARTY,
-                            'packages',
-                            'pyside2-5.15.2.1-py3.10-rev3-windows',
-                            'pyside2',
-                            'lib',
-                            'site-packages')
 
-try:
-    QT_PLUGIN_PATH = QT_PLUGIN_PATH.resolve(strict=True)
-    os.environ[ENVAR_QT_PLUGIN_PATH] = str(QT_PLUGIN_PATH)
-    add_site_dir(QT_PLUGIN_PATH)
-except Exception as e:
-    _LOGGER.warning(f'{ENVAR_QT_PLUGIN_PATH} will not resolve: {QT_PLUGIN_PATH}')
-    _LOGGER.error(f'{e} , traceback =', exc_info=True)
-    if DCCSI_STRICT:
-        _LOGGER.exception(f'{e} , traceback =', exc_info=True)
-        raise e
+if DCCSI_TEST_PYSIDE:
+    try:
+        import azlmbr
+        import azlmbr.bus
+        params = azlmbr.qt.QtForPythonRequestBus(azlmbr.bus.Broadcast, 'GetQtBootstrapParameters')
+        QT_PLUGIN_PATH = Path(params.qtPluginsFolder)
+    except:
+        QT_PLUGIN_PATH = None
+
+    if not QT_PLUGIN_PATH:
+        # fallback, not future proof without editing file
+        # modify to be a grep?
+        # path constructor
+        QT_PLUGIN_PATH = Path(PATH_O3DE_3RDPARTY,
+                                'packages',
+                                'pyside2-5.15.2.1-py3.10-rev3-windows',
+                                'pyside2',
+                                'lib',
+                                'site-packages')
```

Comment:
This should only fail if the `QtForPython` gem wasn't enabled properly, so maybe we should just bail out here if `QT_PLUGIN_PATH` is none?

Reply:
Trying to get external access functional again, we have utils/tools that run outside the editor.

Reply:
Makes sense, sounds good!